### PR TITLE
Add Norwegian NER

### PR DESCRIPTION
**Pull request by jplu**
This PR adds the [Norwegian NER](https://github.com/ljos/navnkjenner) dataset.

I have added the `conllu` package as a test dependency. This is required to properly parse the `.conllu` files.

### CODE REVIEW

File: datasets/norwegian_ner/norwegian_ner.py [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+@inproceedings{johansen2019ner,
+  title={Named-Entity Recognition for Norwegian},
+  author={Johansen, Bjarte},
+  booktitle={Proceedings of the 22nd Nordic Conference on Computational Linguistics, NoDaLiDa},
+  year={2019}
+}
+"""
+
+_DESCRIPTION = """\
+Named entities Recognition dataset for Norwegian.
```
Comment (lhoestq) [inline datasets/norwegian_ner/norwegian_ner.py:34]: Could you add a bit more details :)

File: datasets/norwegian_ner/README.md [hunk 2]
```diff
@@ -0,0 +1,137 @@
+---
+annotations_creators:
+- expert-generated
+language_creators:
+- crowdsourced
+languages:
+- sv
+licenses:
+- other-
```
Comment (lhoestq) [inline datasets/norwegian_ner/README.md:9]: is this other or unknown ? please use the tagger to use valid license types

File: datasets/norwegian_ner/README.md [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+multilinguality:
+- monolingual
+size_categories:
+- 10K<n<100K
+source_datasets:
+- original
+task_categories:
+- other
+task_ids:
+- other-other-Named Entity Recognition
```
Comment (lhoestq) [inline datasets/norwegian_ner/README.md:19]: could you specify named entity recognition here ?

**Review by lhoestq — COMMENTED** [review]
Very cool !

File: datasets/norwegian_ner/README.md (hunk 3 above)
Comment (jplu) [reply datasets/norwegian_ner/README.md:19]: What should be specified more precisely? I don't understand sorry.

File: datasets/norwegian_ner/README.md [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+multilinguality:
+- monolingual
+size_categories:
+- 10K<n<100K
+source_datasets:
+- original
+task_categories:
+- other
+task_ids:
+- Named Entity Recognition
```
Comment (lhoestq) [inline datasets/norwegian_ner/README.md:19]:
```suggestion
task_categories:
- structure-prediction
task_ids:
- named-entity-recognition
```
here you go, sorry if that wasn't clear

**Comment by lhoestq** [comment]
Merging this one, good job and thank you @jplu :)

File: datasets/norwegian_ner/README.md (hunk 2 above)
Comment (lhoestq) [reply datasets/norwegian_ner/README.md]: Haha good catch ! Pinging @jplu

Comment (jplu) [reply datasets/norwegian_ner/README.md]: Good catch indeed 😄 Taking care of this!

**Pull request merged**